scripts/eachdist.py

```diff
@@ -263,7 +263,11 @@ def find_targets_unordered(rootpath):
     for subdir in rootpath.iterdir():
         if not subdir.is_dir():
             continue
-        if subdir.name.startswith(".") or subdir.name.startswith("venv"):
+        if (
+            subdir.name.startswith(".") or 
+            subdir.name.startswith("venv") or 
+            subdir.name.startswith("skiptargetinstall-")
```

Comment:
I thought this was a "consistent with what we've done before" way to get `eachdist.py` to not try to install repos it shouldn't. I think a better fall up would be for this for this script to read a "ignore" configuration from the `eachdist.ini` file.